apps/web/src/components/NftSelector/NftSelector.tsx

```diff
@@ -234,6 +236,19 @@ function NftSelectorInner({ onSelectToken, headerText, preSelectedContract, even
     await syncTokens({ type: filterType, chain: network });
   }, [refreshDisabled, track, eventFlow, filterType, syncTokens, network]);
 
+  const [syncCreatedTokensForExistingContract, isContractRefreshing] =
+    useSyncCreatedTokensForExistingContract();
+  const contractRefreshDisabled = filterType !== 'Created' || isContractRefreshing;
+
+  const handleCreatorRefreshContract = useCallback(async () => {
+    if (!collectionContractId) {
+      return;
+    }
+    track('NFT Selector: Clicked Sync Creator Tokens For Existing Contract');
```

Comment:
same thing, let's make sure to track additional details: `{ id: Refresh Single Contract Button, context: contexts.Posts }`